pkg/acsengine/const.go

```diff
@@ -109,7 +109,7 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"addonresizer": "addon-resizer:2.0",
 		"heapster":     "heapster:v1.4.0",
 		"dns":          "k8s-dns-kube-dns-amd64:1.14.4",
-		"addonmanager": "kube-addon-manager-amd64:v6.1.2",
+		"addonmanager": "kube-addon-manager-amd64:v6.4-beta.2",
```

Comment:
So we need 6.4 and it is still in beta. Is that correct? I guess this is okay, but just makes me kind of nervous to use beta versions.